webapp/channels/src/i18n/en.json

```diff
@@ -2823,9 +2823,9 @@
   "app.channel.post_update_channel_purpose_message.removed": "{username} removed the channel purpose (was: {old})",
   "app.channel.post_update_channel_purpose_message.updated_from": "{username} updated the channel purpose from: {old} to: {new}",
   "app.channel.post_update_channel_purpose_message.updated_to": "{username} updated the channel purpose to: {new}",
-  "app.post.move_thread_command.channel.multiple_messages": "A thread with {{.NumMessages}} messages has been moved: {{.Link}}\n",
-  "app.post.move_thread_command.channel.one_message": "A message has been moved: {{.Link}}\n",
-  "app.post.move_thread_command.direct_or_group.multiple_messages": "A thread with {{.NumMessages}} messages has been moved to a Direct/Group Message\n",
+  "app.post.move_thread_command.channel.multiple_messages": "A thread with {numMessages, number} messages has been moved: {link}\n",
+  "app.post.move_thread_command.channel.one_message": "A message has been moved: {link}\n",
+  "app.post.move_thread_command.direct_or_group.multiple_messages": "A thread with {numMessages, number} messages has been moved to a Direct/Group Message\n",
```

Comment:
These strings were actually completely invalid because they use the Go style of adding values to the string. I don't think that came up before because this feature is experimental, but FormatJS started complaining about them after I changed these to use `defineMessages`